clients/client-core/src/config/mod.rs

```diff
@@ -22,7 +22,10 @@ const DEFAULT_MESSAGE_STREAM_AVERAGE_DELAY: Duration = Duration::from_millis(20)
 const DEFAULT_AVERAGE_PACKET_DELAY: Duration = Duration::from_millis(50);
 const DEFAULT_TOPOLOGY_REFRESH_RATE: Duration = Duration::from_secs(5 * 60); // every 5min
 const DEFAULT_TOPOLOGY_RESOLUTION_TIMEOUT: Duration = Duration::from_millis(5_000);
-const DEFAULT_GATEWAY_RESPONSE_TIMEOUT: Duration = Duration::from_millis(5_500);
+// Set this to a high value for now, so that we don't risk sporadic timeouts that might cause
+// bought bandwidth tokens to not have time to be spent; Once we remove the gateway from the
+// bandwidth bridging protocol, we can come back to a smaller timeout value
+const DEFAULT_GATEWAY_RESPONSE_TIMEOUT: Duration = Duration::from_secs(5 * 60);
```

Comment:
I guess the question is whether the underlying OS won't close the connection without any packets sent on it...

Reply:
If there are signs that this happens (I don't know exactly the timeouts for different OSes), we can lower the Ethereum block depth we require, from the current value of 7 to something else. On my machine, there is no timeout even in release mode, which awaits ~120s for the event.

Reply:
We can't really lower it as 7 is already a bare minimum for some safety.
The timeout I was talking about is something I've been experiencing with the network monitor -> the gateway connection would time out after a while if no messages were sent on it.